src/store/mutationTypes.js

```diff
@@ -3,3 +3,4 @@ export const SAVE_CURRENT_WEATHER = 'SAVE_CURRENT_WEATHER';
 export const SAVE_ERROR_DESC = 'SAVE_ERROR_DESC';
 export const SAVE_CURRENT_POSITION_DATA = 'SAVE_CURRENT_POSITION_DATA';
 export const SAVE_HOURLY_FORECAST = 'SAVE_HOURLY_FORECAST';
+export const SAVE_PREFER_DARK_THEME = 'SAVE_PREFER_DARK_THEME';
```

Comment:
[Optional] Call it `SAVE_PREFERRED_THEME` (and pass `dark` as an argument later). In this case it's much easier to add new theme